Sources/Control/Volume.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Volume.swift
+//  ControlKit
+//
+
+import MediaPlayer
+
+public extension Control {
+    
+    /// Control the system volume using the underlying `MPVolumeView`.
+    @MainActor
+    enum Volume {
+        
+        /// Subscribe to `volume` via ``VolumeController.volume`` `@Published` property.
+        public static var volume: Float {
+            get { MPVolumeView.volume }
+            set { MPVolumeView.volume = newValue }
+        }
+        
+        /// Subscribe to `isMuted` via ``VolumeController.isMuted`` `@Published` property.
+        static var isMuted = false
```

Comment:
I guess, `didSet` or `private(set)` is needed here 👀

Reply:
Using `didSet` instead of the extra `toggleMute()` is much more elegant 💅

Also updated in the `fixup` commit.